### PR TITLE
feat(react-hoc-conversation-mercury): refactor mercury conversation events

### PR DESCRIPTION
**Pull request by adamweeks**
✅ Automation tests for widget-space pass

### CODE REVIEW

File: packages/node_modules/@ciscospark/widget-message/src/container.js [hunk 1]
```diff
@@ -294,20 +294,18 @@ export class MessageWidget extends Component {
...
-    props.updateMercuryState({isListeningToActivity: true});
+    props.updateWidgetState({isListeningToActivity: true});
     sparkInstance.internal.mercury.on(`event:conversation.activity`, (event) => {
       const activity = event.data.activity;
       const isSelf = activity.actor.id === props.user.get(`currentUser`).id;
       // Ignore activity from other conversations
       if (activity.target && activity.target.id === space.id) {
         if (activity.object.objectType === `activity`) {
-          props.receiveMercuryActivity(activity);
           if (isSelf && activity.verb === `acknowledge`) {
```
Comment (bzang) [inline packages/node_modules/@ciscospark/widget-message/src/container.js]: Can we combine the 2 sequential if statements into a single one?

File: packages/node_modules/@ciscospark/widget-message/src/container.js [hunk 2]
```diff
@@ -722,14 +720,12 @@ export default connect(
...
     retryFailedActivity,
     setScrollPosition,
     setTyping,
     showScrollToBottomButton,
     updateHasNewMessage,
-    updateMercuryState
+    updateWidgetState
   }, dispatch)
-)(MessageWidget);
+)(ConnectedMessageWidget);
```
Comment (bzang) [inline packages/node_modules/@ciscospark/widget-message/src/container.js]: Is this the right order of applying `connect()`? I can't tell if this is right or the order should be reversed like:
``` js
const MessageWidget = connect(getMessageWidgetProps, blah blah)(MessageWidget);
export default wrapConversationMercury(MessageWidget);

File: packages/node_modules/@ciscospark/react-hoc-conversation-mercury/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@ciscospark/react-hoc-conversation-mercury",
+  "version": "0.1.0",
```
Comment (bzang) [inline packages/node_modules/@ciscospark/react-hoc-conversation-mercury/package.json]: Let's remove versions from the packages `package.json` files. They don't mean anything, but seem to have caused some confusion for devs.

File: packages/node_modules/@ciscospark/widget-message/src/container.js (hunk 2 above)
Comment (adamweeks) [reply packages/node_modules/@ciscospark/widget-message/src/container.js]: I would think we would need to connect the wrapped component so it gets all the redux stores it needs.

File: packages/node_modules/@ciscospark/widget-message/src/container.js (hunk 1 above)
Comment (adamweeks) [reply packages/node_modules/@ciscospark/widget-message/src/container.js]: Oh yea, will do.

**Pull request merged**